doc/nrf/releases/release-notes-2.0.0.rst

```diff
@@ -0,0 +1,1091 @@
+      * ``zcl cmd`` shell command extended to allow sending groupcasts.
+      *  ``zdo`` shell commands extended to allow binding to a group address.
+      * Internal context manager structures.
+
+    * Fixed:
+
+      * The issue where the `zcl cmd` shell command was using the incorrect index of a context manager entry during cleanup after the command was sent.
+
+  * :ref:`lib_zigbee_zcl_scenes`:
+
+    * Updated the library, so that it is allowed to store empty scenes.
+
+  * :ref:`lib_zigbee_osif`:
+
+    * Updated:
+
+      * Crypto library used for performing software AES ecryption.
+        Now, the :ref:`nrfxlib:nrf_oberon_readme` is used instead of the Tinycrypt library.
+      * Optimize calling ZBOSS API in |NCS| platform.
+        If the ZBOSS API is called in the ZBOSS thread context, processing by the workqueue is now skipped.
+
+  * :ref:`lib_zigbee_fota`:
+
+    * Added:
+
+      * New :kconfig:option:`CONFIG_ZIGBEE_FOTA_SERVER_DISOVERY_INTERVAL_HRS` Kconfig option to configure the interval between queries for the Zigbee FOTA server.
+      * New :kconfig:option:`CONFIG_ZIGBEE_FOTA_IMAGE_QUERY_INTERVAL_MIN` Kconfig option to configure the interval between queries for the available Zigbee FOTA images.
+      * Support for the combined application and network core updates for the nRF5340 SoC.
+
+    * Updated:
+
+      * Download logic to use the :ref:`lib_dfu_multi_image` library API and image structure.
+      * The image generation script by introducing the sub-element structure inside the Zigbee OTA image.
+        Enable :kconfig:option:`CONFIG_ZIGBEE_FOTA_GENERATE_LEGACY_IMAGE_TYPE` to generate images compatible with previous NCS releases.
+      * Default value of the :kconfig:option:`CONFIG_ZIGBEE_FOTA_IMAGE_TYPE` to 0x0141.
+      * Setting of the :kconfig:option:`CONFIG_NRF53_ENFORCE_IMAGE_VERSION_EQUALITY` for nRF5340 SoC to ensure integrity of the upgrade image.
+
+    * Removed:
+
+      * The :c:enum:`ZIGBEE_FOTA_EVT_ERASE_PENDING` and :c:enum:`ZIGBEE_FOTA_EVT_ERASE_DONE` events.
+
+  * Fixed:
+
+    * An issue where printing binding table containing group-binding entries results in corrupted output.
+    * An issue where Zigbee shell coordinator would not form a new network after the factory reset operation.
+
+
+sdk-nrfxlib
+-----------
+
+See the changelog for each library in the :doc:`nrfxlib documentation <nrfxlib:README>` for additional information.
+
+Scripts
+=======
+
+This section provides detailed lists of changes by :ref:`script <scripts>`.
+
+* Added new ``west ncs-sbom`` command that generates :ref:`Software Bill of Materials <west_sbom>`.
+* Added :ref:`bt_fast_pair_provision_script`.
+
+MCUboot
+=======
+
+The MCUboot fork in |NCS| (``sdk-mcuboot``) contains all commits from the upstream MCUboot repository up to and including ``e86f575f68fdac2cab1898e0a893c8c6d8fd0fa1``, plus some |NCS| specific additions.
```

Comment:
The commit has to be verified.

Reply:
@carlescufi  @nvlsianpu  please verify MCUboot commit

Reply:
The SHA is correct